library/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
 
         <provider
             android:name=".provider.TrayContentProvider"
-            android:authorities="@string/tray__authority"
+            android:authorities="${applicationId}.tray"
```

Comment:
I guess Instant Run will not work with AndroidManifest Placeholders :(
They think that the Manifest have changed and don't run it in "Instant Run Mode"...